examples/models/sklearn_spacy_text/sklearn_spacy_text_classifier_example.ipynb

```diff
@@ -45,19 +45,85 @@
   },
```

Comment:
<p>Would be worth clearing the logs for long outputs like this one</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/1935/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
should be fixed now